### DISCUSSION TITLE
Need 0.01 eth on mainnet

### DISCUSSION THREAD

**Discussion by fraanciisq** [Cyfrin/foundry-full-course-cu#2988] [General]
I tried all the links for getting sepolia eth testnet but getting error and still needs 0.01 eth in mainnet

**Comment by EngrPips** [comment]
Have you tried this https://cloud.google.com/application/web3/faucet/ethereum/sepolia

**Reply by fraanciisq** [reply]
Yes still the same

**Reply by fraanciisq** [reply]
It generates this prompt: To help prevent automated abuse, please verify that your wallet address has at least 0.001 ETH on Ethereum Mainnet

**Reply by EngrPips** [reply]
Getting Sepolia ETH has been tough lately. I will advise you to test things locally and keep retrying the faucet. It seems to always work if you retry after few minutes

**Reply by uzairnadeem201** [reply]
from this https://cloud.google.com/application/web3/faucet/ethereum/sepolia i got the sepolia etherscan is showing successfull transaction as well but in my meta wallet theres still 0 sepoliaeth any solution?

**Reply by EngrPips** [reply]
Are you sure you are checking the correct Metamask account?

**Reply by khandelwalmoksh** [reply]
@fraanciisq use this https://chainlist.org/?testnets=true
